cla_frontend/assets-src/javascripts/app/js/controllers/eligibility_check.js

```diff
@@ -346,8 +351,13 @@
             // if not SMOD, set to not disputed
             if (!$scope.hasSMOD()) {
               property.disputed = 0;
+
             }
 
+            $scope.$evalAsync(function() {
+              $scope.isPropertyAdded = true;
```

Comment:
I can't see any usage of this property, what's its purpose?

Reply:
Apologies that is old code, will remove it now